secure.go

```diff
@@ -58,14 +63,20 @@ type Options struct {
 	// CustomBrowserXssValue allows the X-XSS-Protection header value to be set with a custom value. This overrides the BrowserXssFilter option. Default is "".
 	CustomBrowserXssValue string
 	// ContentSecurityPolicy allows the Content-Security-Policy header value to be set with a custom value. Default is "".
+	// Passing a template string will replace `{{ . }}` with a dynamic nonce value for each request which can be later retrieved using the Nonce function.
+	// Eg: script-src {{ . }} -> script-src 'nonce-a2ZobGFoZg=='
 	ContentSecurityPolicy string
+	// CSPNonceByteAmount is the byte size of the base64 nonce string being generated default is 16.
```

Comment:
Is there a use-case for configuring the number of bytes here?
It seems easier to just pick a number which is sufficient for all use-cases (16 is probably good enough), and fewer knobs is generally better.

Reply:
Strongly agree. I don't think there's a need to make this configurable: provide a 16-byte nonce by default + use crypto/rand.